dot/sync/chain_sync_integeration_test.go

```diff
@@ -205,34 +171,34 @@ func TestChainSync_sync_tip(t *testing.T) {
 	require.Equal(t, tip, cs.state)
 }
 
-func TestChainSync_getTarget(t *testing.T) {
+func TestChainSync_getTarget_Integration(t *testing.T) {
 	cs, _ := newTestChainSync(t)
-
+	require.Equal(t, uint(1<<32-1), cs.getTarget())
 	cs.peerState = map[peer.ID]*peerState{
 		"a": {
 			number: 0, // outlier
 		},
 		"b": {
-			number: 110,
+			number: 10,
 		},
 		"c": {
-			number: 120,
+			number: 20,
 		},
 		"d": {
-			number: 130,
+			number: 30,
 		},
 		"e": {
-			number: 140,
+			number: 40,
 		},
 		"f": {
-			number: 150,
+			number: 50,
 		},
 		"g": {
 			number: 1000, // outlier
 		},
 	}
 
-	require.Equal(t, uint(130), cs.getTarget()) // sum:650/count:5 = avg:130
+	require.Equal(t, uint(25), cs.getTarget()) // sum:150/count:6= avg:25
```

Comment:
Why do we have different values here?

Reply:
I don't know.

Reply:
Please revert the changes made for this unit test.

Reply:
Reverted.